sdks/java/core/src/main/java/org/apache/beam/sdk/runners/inprocess/InProcessBundleFactory.java

```diff
@@ -73,23 +70,19 @@ private InProcessBundleFactory() {}
      * Create a new {@link InProcessBundle} for the specified {@link PCollection} without a key.
      */
     public static <T> InProcessBundle<T> unkeyed(PCollection<T> pcollection) {
-      return new InProcessBundle<T>(pcollection, false, null);
+      return new InProcessBundle<T>(pcollection, null);
     }
 
     /**
      * Create a new {@link InProcessBundle} for the specified {@link PCollection} with the specified
      * key.
-     *
-     * <p>See {@link CommittedBundle#getKey()} and {@link CommittedBundle#isKeyed()} for more
-     * information.
      */
     public static <T> InProcessBundle<T> keyed(PCollection<T> pcollection, Object key) {
```

Comment:
Yes, but this was always the case (and `CommittedBundle#getKey()` is annotated `@Nullable`)

Annotated the factory method, field, and other factory method.